go/client/fuse_status_osx.go

```diff
@@ -0,0 +1,86 @@
+// +build darwin
+
+package client
+
+import (
+	"fmt"
+	"os"
+
+	keybase1 "github.com/keybase/client/go/protocol"
+	"github.com/keybase/go-kext"
+)
+
+func KeybaseFuseStatus(bundleVersion string) keybase1.FuseStatus {
+	status := keybase1.FuseStatus{
+		BundleVersion: bundleVersion,
+		InstallStatus: keybase1.InstallStatus_UNKNOWN,
+		InstallAction: keybase1.InstallAction_UNKNOWN,
+	}
+
+	var kextInfo *kext.Info
+
+	// Check osxfuse 2.x
+	path2 := "/Library/Filesystems/osxfusefs.fs"
```

Comment:
can these 2 blocks share code?  Seems like a lot of code duplication between the `2.x` and `3.x` cases.